lib/cli/src/commands/whoami.rs

```diff
@@ -0,0 +1,18 @@
+use clap::Parser;
+
+#[derive(Debug, Parser)]
+/// The options for the `wasmer whoami` subcommand
+pub struct Whoami {
+    /// Which registry to check the logged in username for
+    #[clap(long, name = "registry")]
+    pub registry: Option<String>,
+}
+
+impl Whoami {
+    /// Execute `wasmer whoami`
+    pub fn execute(&self) -> Result<(), anyhow::Error> {
+        let (registry, username) = wasmer_registry::whoami(self.registry.as_deref())?;
+        println!("logged into registry {registry:?} as user {username:?}");
```

Comment:
Can you switch this over to a `log` statement?

Reply:
In this case it needs to be always printed, with log it would prefix a `LOG ...` statement which I'd really not want

Reply:
Good point!